### PR TITLE
New subscriptions design

### PR DESCRIPTION
**Pull request by ClementPasteau**
Full design is incomplete, but it's already a good step forward.
Storybook: https://gdevelop-storybook.s3.amazonaws.com/new-subscriptions-design/latest/index.html?path=/story/subscription-subscriptiondetails--loading-subscription

image
image
image
image
image

### CODE REVIEW

File: newIDE/app/src/ProjectCreation/NewProjectSetupDialog.js [hunk 1]
```diff
@@ -628,7 +628,9 @@ const NewProjectSetupDialog = ({
                 limits={limits}
```
Comment (AlexandreSi) [inline newIDE/app/src/ProjectCreation/NewProjectSetupDialog.js:628]: Should this also be renamed quota?

Comment (AlexandreSi) [reply newIDE/app/src/ProjectCreation/NewProjectSetupDialog.js:628]: Oh no, I think I confused the two concepts

File: newIDE/app/src/Profile/Subscription/PlanCard.js [hunk 2]
```diff
@@ -67,62 +62,117 @@ const getPlanPrice = ({
...
+      return <Silver fontSize="large" style={{ width: 65, height: 65 }} />;
+    case 'gdevelop_gold':
+    case 'gdevelop_pro': // legacy
+      return <Gold style={{ width: 65, height: 65 }} />;
+    // TODO: Add icons for other plans.
+    case 'gdevelop_education':
+    case 'gdevelop_startup':
+    case 'gdevelop_enterprise':
+    default:
+      return <GDevelopGLogo style={{ width: 25, height: 25, padding: 20 }} />;
```
Comment (AlexandreSi) [inline newIDE/app/src/Profile/Subscription/PlanCard.js:78]: Should those values be stored in constants with an operation like `WIDTH - 2*PADDING`?

File: newIDE/app/src/Profile/Subscription/SubscriptionDetails.js [hunk 3]
```diff
@@ -168,11 +242,11 @@ const SubscriptionDetails = ({
               <I18n>
                 {({ i18n }) => (
                   <Paper background="dark" variant="outlined">
-                    <LineStackLayout alignItems="center">
+                    <LineStackLayout alignItems="center" noMargin>
                       <img
                         src="res/diamond.svg"
                         style={styles.diamondIcon}
-                        alt=""
+                        alt="diamond"
```
Comment (AlexandreSi) [inline newIDE/app/src/Profile/Subscription/SubscriptionDetails.js:249]: 💜

**Review by AlexandreSi — APPROVED** [review]
LGTM

File: newIDE/app/src/Profile/Subscription/PlanCard.js (hunk 2 above)
Comment (ClementPasteau) [reply newIDE/app/src/Profile/Subscription/PlanCard.js:78]: yep, makes sense, will do

**Pull request merged**